app/org/sagebionetworks/bridge/models/studies/ConsentSignature.java

```diff
@@ -6,17 +6,27 @@
 import javax.annotation.Nullable;
 
 import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
 import org.sagebionetworks.bridge.json.JsonUtils;
 import org.sagebionetworks.bridge.models.BridgeEntity;
 import org.sagebionetworks.bridge.validators.ConsentSignatureValidator;
 import org.sagebionetworks.bridge.validators.Validate;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFilter;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
 
+@JsonFilter("filter")
 public final class ConsentSignature implements BridgeEntity {
 
+    public static final ObjectWriter SIGNATURE_WRITER = new BridgeObjectMapper().writer(
+            new SimpleFilterProvider().addFilter("filter", 
+            SimpleBeanPropertyFilter.serializeAllExcept("signedOn")));
```

Comment:
We could include it, adding a field shouldn't break anything. However, if we do, we'll have to serialize it as an ISO datetime stamp and that would make it a bit harder to match to the long value in the existing user consent table. So, not adding it to the REST API and allowing it to serialize as a number in Stormpath seemed easier.